canonical/ubuntu.com
# [WD-5489] replace an image with HTML table on Ceph page

## Done

- Replaced an image that showed a table with HTML table
- Created a SCSS pattern styles for HTML bordered table for future similar use cases (and to use SCSS variables from Vanilla)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Naviage to `/ceph` page and scroll down to `Benefits of Ceph on Ubuntu` and test the table
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Closes https://warthogs.atlassian.net/browse/WD-5489

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
